kpconv_torch/io/las.py

```diff
@@ -2,7 +2,7 @@
 import numpy as np
 
 
-def read_las_laz(filepath):
+def read_las_laz(filepath, xyz_only=False):
     """Takes a file path pointing on a 3D point .las or .laz file and returns the points,
     the associated colors and the associated classes.
 
```

Comment:
Add parameter in the docstring

Reply:
Well the parameter is added in the docstring, vut it is in the [numpydoc style](https://numpydoc.readthedocs.io/en/latest/format.html#parameters).

Anyway, I will modify this in order to consider the [Sphinx style](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#the-sphinx-docstring-format), as most of the docstrings in the source code use this standard.